tests/test_integrated_channels/test_utils.py

```diff
@@ -277,56 +277,56 @@ def test_is_url_valid(self):
     def test_get_courserun_duration_in_hours(self, course_run, expected_duration_days):
         assert utils.get_courserun_duration_in_hours(course_run) == expected_duration_days
 
-    def test_truncate_item_collections(self):
-        in_a = [1, 2, 3]
-        in_b = [1, 2, 3, 4, 5]
-        in_c = [1, 2, 3, 4, 5, 6, 7]
+    def test_truncate_item_dicts(self):
```

Comment:
because we're truncating later in the workflow, we are now working with `dict` data structures instead of `list` - this truncate needed to be reworked and retested